android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerPackage.java

```diff
@@ -51,9 +52,9 @@ public void updateExtraData(View root, Object extraData) {
     public @Nullable Map getExportedCustomDirectEventTypeConstants() {
       return MapBuilder.of(
               RNGestureHandlerEvent.EVENT_NAME,
-              MapBuilder.of("registrationName", RNGestureHandlerEvent.EVENT_NAME),
+              MapBuilder.of("registrationName", "onGestureHandlerEvent"),
```

Comment:
Can you put both "topGestureHandlerEvent" and "onGestureHandlerEvent" as strings here instead of using constant? Or maybe define "onXXX" as a constant in the event class? I think it will make it easier to understand how events are translated in the future as well as may help with spotting typos etc

(same below)